docs/spicedb/enabling-watch-api.md

```diff
@@ -0,0 +1,25 @@
+# Enabling Watch API
```

Comment:
I'm not sure this should be in its own article, but rather a section in the data stores doc

Reply:
I want it distinct for easy linking

Reply:
Except it makes it less discoverable. If you are reading the datastore docs you'd have no idea that you'd have to follow instructions on another doc. Maybe if we add links from there to here, that'd be sufficient

Reply:
I'll add links